_env/README.md

```diff
@@ -1,67 +1,28 @@
-# Deployment
+# Deploying the Website
 
 ## Setting up
 
-To setup, you need to have the following installed:
-
-* [GCloud][gcloud]
-
-You also need to have installed `kubectl`, which powers Kubernetes, using the
-following command:
-
-```bash
-gcloud components install kubectl
-```
-
-This also expects you've spun up a Google Container Engine (GKE) cluster,
-similar to the one we use in the Student Robotics Stack.
-
-Set the project you'll be working on
-
-```bash
-gcloud config set project main-site-141811
-```
-
-## Logging into Kubernetes
-
-```bash
-gcloud container clusters get-credentials srobo-website --region europe-west1-b
-```
-
-## HTTPS certificates
-
-The application needs a HTTPS certificate and private key to be in place,
-otherwise the app won't start. We use Kubernetes secret management for this.
-
-```bash
-kubectl create secret generic https-keys \
-    --from-file=cert.pem \
-    --from-file=key.pem
-```
+This assumes you've got access to a kubernetes cluster. We have [a guide][guide]
+on how we do this at Student Robotics.
 
 ## Deploy the application
 
-Todo: Template this somehow
-
-Within `kubernetes/service.yml`, you need to replace the static IP we use
-(i.e. `104.155.107.130`) with the one you intend to use to expose your
-application. Then you run:
-
 ```bash
 kubectl create -f kubernetes/
```

Comment:
Is this line still useful? (I'm assuming this is part of the initial deployment -- it's not something I've ever run to deploy updates)

Reply:
It depends if you've changed any of the config in that directory.

Reply:
Ah, that might be worth saying explicitly?

Reply:
Turns out the command is subtly different when updating an existing deployment in Kubernetes, I've added this to a new commit.

Reply:
https://github.com/srobo/website/pull/155/commits/b4da306d387d209967dbc3f77407b0f4d3b13d23